winter/pagination/page_position_argument_resolver.py

```diff
@@ -19,7 +20,14 @@
 
 class PagePositionArgumentResolver(ArgumentResolver):
 
-    def __init__(self, limit_name: str = 'limit', offset_name: str = 'offset', order_by_name: str = 'order_by'):
+    def __init__(
+        self,
+        limit_name: str = 'limit',
+        offset_name: str = 'offset',
+        order_by_name: str = 'order_by',
+        allow_any_order_by_field: bool = False
```

Comment:
Trailing comma? Don't we have flake8 configured for this?

Reply:
No, we do not use it ((